src/op_handlers/revert.rs

```diff
@@ -63,7 +61,8 @@ fn revert_far_call(vm: &mut VMState) -> Result<(), EraVmError> {
     Ok(())
 }
 
-pub fn revert_out_of_gas(vm: &mut VMState) -> Result<(), EraVmError> {
+pub fn revert_out_of_gas(vm: &mut VMState, storage: &mut dyn Storage) -> Result<(), EraVmError> {
+    storage.rollback(&vm.current_frame()?.storage_before.clone());
```

Comment:
Rolling back on a clone doesn't make a lot of sense to me. Why not just pass by reference?

Reply:
Yes that would be fine